source/includes/api/_user_management.md

```diff
@@ -57,65 +85,132 @@ curl -X POST http://api.resourcewatch.org/auth/login \
 }
 ```
 
+Endpoint for email + password based login.
+
+For HTML requests, it will redirect to either `<BASE API URL>/auth/success` or `<BASE API URL>/auth/fail` depending on whether the login was successful or not. If successful, the HTTP reply will have a session cookie that may be used in subsequent requests. An optional `callbackUrl` query parameter can be provided, in which case the user will be redirected to that URL in case of login success. 
+
+For JSON requests, in case of successful logins, the user details will be returned as a JSON object.
+
+**Errors**
+
+Error code     | Error message  | Description
+-------------- | -------------- | --------------
+401            | Invalid email or password. | Login credentials are missing or incorrect.
+
 ### GET `<BASE API URL>/auth/fail`
+
 Displays login errors for HTML requests. Not supported on JSON requests.
 
 ### GET `<BASE API URL>/auth/check-logged`
-Check login status
+
+> Check if the user is logged
+
+```shell
+curl -X GET "https://api.resourcewatch.org/auth/check-logged" \
+    -H 'Cookie: <your cookie values>'
+```
+
+> Response
+
+```json
+{
+  "data": {
+    "id": "5bfd237767b3176dd63f2eb7",
+    "email": "your-email@provider.com",
+    "createdAt": "2018-11-27T10:59:03.531Z",
+    "role": "USER",
+    "extraUserData": {
+      "apps": ["rw"]
+    }
+  }
+}
+```
+
+Checks if you are logged in. This is typically used to determine if a session has been established between the the user's browser and the RW API. 
+
+**Errors**
+
+Error code     | Error message  | Description
+-------------- | -------------- | --------------
+401            | Not authenticated. | You are not logged in.
 
 ### GET `<BASE API URL>/auth/success`
+
 Successful login page for HTML requests. Not supported on JSON requests.
 
 ### GET `<BASE API URL>/auth/logout`
-Login invalidation endpoint
+
+Login invalidation endpoint. Only invalidates the session cookie set on login. If using JWT token based authentication, this endpoint will NOT invalidate the token.
 
 ### GET `<BASE API URL>/auth/generate-token`
-Generates a JWT token for the current user session.
+
+> Generate the user's JWT token
+
+```shell
+curl -X GET "https://api.resourcewatch.org/auth/generate-token" \
+    -H 'Cookie: <your cookie values>'
+```
+
+> Response:
+
+```json
+{
+    "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjViZWNmYTJiNjdkYTBkM2VjMDdhMjdmNiIsInJvbGUiOiJVU0VSIiwicHJvdmlkZXIiOiJsb2NhbCIsImVtYWlsIjoidGVzdEBleGFtcGxlLmNvbSIsImV4dHJhVXNlckRhdGEiOnsiYXBwcyI6WyJydyJdfSwiY3JlYXRlZEF0IjoxNTQzMzE1NzMxNzcwLCJpYXQiOjE1NDMzMTU3MzF9.kIdkSOb7mCMOxE2ipqVOBrK7IefAjLDhaPG9DT1qvCw"
```

Comment:
This isn’t anyone’s actual token, right?

Reply:
No, it's a specially crafted for these docs only, and doesn't match any existing user